### PR TITLE
Regen with gtk 3.99.5

### LINKED ISSUE #40: TreeStore: manually implement remove
the param uses `inout` direction which can't be automatically generated by gir for now, this needs to be manually implemented for now.

```
  <method name="remove" c:identifier="gtk_tree_store_remove">
        <doc xml:space="preserve">Removes @iter from @tree_store.  After being removed, @iter is set to the
next valid row at that level, or invalidated if it previously pointed to the
last one.</doc>
        
        <return-value transfer-ownership="none">
          <doc xml:space="preserve">%TRUE if @iter is still valid, %FALSE if not.</doc>
          <type name="gboolean" c:type="gboolean"/>
        </return-value>
        <parameters>
          <instance-parameter name="tree_store" transfer-ownership="none">
            <doc xml:space="preserve">A #GtkTreeStore</doc>
            <type name="TreeStore" c:type="GtkTreeStore*"/>
          </instance-parameter>
          <parameter name="iter" direction="inout" caller-allocates="0" transfer-ownership="full">
            <doc xml:space="preserve">A valid #GtkTreeIter</doc>
            <type name="TreeIter" c:type="GtkTreeIter*"/>
          </parameter>
        </parameters>
      </method>
```

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Replaces #75 

Closes #40

### CODE REVIEW

File: gtk4/Gir.toml [hunk 1]
```diff
@@ -1021,12 +959,6 @@ trust_return_value_nullability = false
     name = "add_choice"
     ignore = true  #strange type of options and option_labels
 
-[[object]]
-name = "Gtk.FileChooserButton"
```
Comment (sdroege) [inline gtk4/Gir.toml:1025]: The file for this also needs to be removed or not? Also check if there are other files :)

**Comment by sdroege** [comment]
Looks ok otherwise, please update gir-files :)

File: gtk4/Gir.toml (hunk 1 above)
Comment (alatiera) [reply gtk4/Gir.toml:1025]: Yes, `FileChooserButton` no longer exists iirc

Comment (sdroege) [reply gtk4/Gir.toml:1025]: Yes I mean the generated rs source file has to be removed from git, if that didn't happen already

Comment (bilelmoussaoui) [reply gtk4/Gir.toml:1025]: ah yes, i thought i did.

**Review by sdroege — APPROVED** [review]

**Comment by bilelmoussaoui** [comment]
CI is happy :yay:

**Pull request merged**